contributing/deployment-tools.txt

```diff
@@ -0,0 +1,132 @@
+
+-  https://github.com/ome/omero-install
+
+
+Testing workflow
+----------------
+
+Devspace (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Continuous integration tools managed by Jenkins CI providing
+an automation framework that runs repeated jobs. The default deployment
+initializes a Jenkins CI master with a predefined set of jobs.
+
+The Devspace repository is available from:
+
+-  https://github.com/openmicroscopy/devspace
+
+Devspace Dockerfiles uses common devslave image. The Devslave repository
+is available from:
+
+-  https://github.com/openmicroscopy/devslave-c7-docker
+
+
+Continuous Delivery (Production)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Production Continuous Delivery (CD) platform managed by
+`Jenkins <http://jenkins-ci.org>`_ available on http://ci.openmicroscopy.org.
+More details about CI-master available on :doc:`continuous-integration`
+
+
+How to add new/upgrade prerequisites
+------------------------------------
+
+When the OME platform requires a new set of prerequisites all the above
+listed repositories may require updates. Depends on the nature of 
+packages developers must consider:
+
+-  adding new Ansible role or complete Ansible playbook to Infrastructure
+
+-  adding new scripts installing appropriate package and its dependencies
+   to OMERO-install, that includes:
+
+     - updating Linux and Mac installation scripts
+     - updating documentation autogen
+- deploying Devspace to test OMERO-install scripts, that includes:
+     - adding new Docker container if requires to support additional processes
+     - adjusting predefined Jenkins jobs
+
+After successful testing new prerequisites can be proposed as a permanent
+adjustment to production CD.
+
+.. note::
+
+    Any Python module that is distributed from Linux distro packages
+    must be installed from RPM file. Python modules only available
+    on PyPI should be added as PIP requirement.
+
+
+Pre release testing
+-------------------
+
+It is also very important to test all the dependencies before release
+to make sure sysadmin instructions are fully tested. The easiest way to
+test is to use Devspace.
+
+Initial testing can be done by uncommenting::
+
+    ## update omero-install to the latest one
+
+on each of the Dockerfile. This section of Docker file should not be
+left uncommented in main repository.
+
+
+Post release
+------------
+
+After adding new prerequisites to Infrastructure and OMERO-install,
+both repositories should be tagged. Tag should be propageted first to
+Devslave docker image and tagged as the following version number.
+New Devslave tag should be then poropagated to all Devspace Docker images.
+
+
+EXAMPLE
+-------
+
+-  adding `Ansible role <https://github.com/openmicroscopy/infrastructure/pull/56>`
+-  adding new scripts to OMERO-install, that includes
+     - `UNIX PR <https://github.com/ome/omero-install/pull/95>`
+       tested via `travis <https://travis-ci.org/ome/omero-install/builds/124104512>`
```

Comment:
I'm just worried about linking to a historical travis-ci build, are they guaranteed to last forever? I think the script is more informative than the logs, especially since everyone already knows about travis-ci